config/bidderinfo.go

```diff
@@ -241,17 +241,11 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 	for fileName, data := range bidderConfigs {
 		bidderName := strings.Split(fileName, ".")
 		if len(bidderName) == 2 && bidderName[1] == "yaml" {
-			normalizedBidderName, bidderNameExists := normalizeBidderName(bidderName[0])
-			if !bidderNameExists {
-				return nil, fmt.Errorf("error parsing config for bidder %s: unknown bidder", fileName)
-			}
 			info := BidderInfo{}
 			if err := yaml.Unmarshal(data, &info); err != nil {
 				return nil, fmt.Errorf("error parsing config for bidder %s: %v", fileName, err)
 			}
 
-			bidderInfos[string(normalizedBidderName)] = info
-
 			//need to maintain nullable fields from BidderInfo struct into bidderInfoNullableFields
 			//to handle the default values in aliases yaml
 			if len(info.AliasOf) > 0 {
```

Comment:
Just to confirm. `info.AliasOf` is not yet set for any alias, correct? 

For example bidders `Adsyield`, `Evtech`, `Iionads`, `Xtrmqb` are aliases of `LimelightDigital`. 

Will you add this new property `aliasOf` to each of them (except of LimelightDigital), for instance file `adsyield.yaml` will look like
```yaml 
endpoint: "http://ads-pbs.open-adsyield.com/openrtb/{{.PublisherID}}?host={{.Host}}"
aliasOf: "limelightDigital"
``` 
So far local testing looks good, I get `feature is currently under development` message, but I see data is loaded properly to `bidderInfos`

Will it be a separate PR for yaml files updates?